### PR TITLE
Blog how to authenticate svelte apps

### PR DESCRIPTION
**Pull request by FuzzySid**
#### Please note that in case of below checklist not updated accordingly, the maintainers have the right to immediately close your pull request.

- [x] You created a branch from `dev` branch.
- [x] You PR is raised on `dev` branch and not on`master`.
- [x] Blog follows [Blog Guidelines](https://github.com/LoginRadius/engineering-portal/blob/master/GUIDELINES.md).
- [x] You have read the [Contribution Guidelines](https://github.com/LoginRadius/engineering-portal/blob/master/CONTRIBUTING.md) before creating this PR.
- [x] Checked for any duplicate blog entry.
- [x] Checked for all grammatical errors in the blog.
- [x] You have not published the blog content anywhere else on the internet on platforms like medium, dev.to etc.
- [x] The content is not copy-pasted from anywhere else on the Internet.

### CODE REVIEW

**Comment by raghunath-r-a** [comment]
Hi @FuzzySid We're moving the PR to Phase - II.

Please add sample code in the following repo by creating a separate pull request: https://github.com/LoginRadius/engineering-blog-samples

Accordingly, we can update GitHub links in lines 87 and 123

**Comment by FuzzySid** [comment]
Hi @raghunath-r-a  Thanks for letting me know, I have done the needful. [Here's](https://github.com/LoginRadius/engineering-blog-samples/pull/157) the pull request for the code sample.

**Review by Akash-Patil — CHANGES_REQUESTED** [review]
Hi @FuzzySid,

I have reviewed the blog and suggested a couple of changes. Please have a look.
Overall, the blog is well-written and informative. 
However, there is a small issue with the demo code as mentioned below.

Right now the submodule for the svelte app is not viewable on the Github UI. In this case, it would be better to use a normal folder for the code to maintain uniformity in the blog-samples repo. Kindly resend a PR with the code in a similar way as to the other demo code examples in the repo. 

Regards,
@Akash-Patil

File: content/guest-post/authenticating-svelte-apps/index.md [hunk 1]
```diff
@@ -0,0 +1,838 @@
...
+       let loginFields={email,password};
+	   console.log(loginFields)
+    }
+    const navigateToSignup=()=>{}
+</script>
+```
+
+At the moment, you're simply encapsulating the `email` and `password` bindings into a JavaScript object. Later, you'll send a request to your Login API here. You also have an empty `navigateToSignup` function that I'll come back to once you set up your routes.
+
+Let's render the Login component inside your `App.svelte` file:
```
[the remark this answers is not in the document]
Comment (FuzzySid) [reply content/guest-post/authenticating-svelte-apps/index.md:217]: Hi @Akash-Patil 
Thanks for letting me know. I have created a new PR for the code sample [here](https://github.com/LoginRadius/engineering-blog-samples/pull/164). I believe that should do the job. Let me know once the PR is merged so I can also update the blog accordingly.

[the remark this answers is not in the document]
Comment (raghunath-r-a) [reply content/guest-post/authenticating-svelte-apps/index.md:217]: Hi @FuzzySid The code sample PR has been merged

[the remark this answers is not in the document]
Comment (FuzzySid) [reply content/guest-post/authenticating-svelte-apps/index.md:217]: I have updated the links in the blog. 
@raghunath-r-a @Akash-Patil

**Review by Akash-Patil — APPROVED** [review]

**Pull request merged**